src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+
+$arrows-color: #000;
+$num-color: #fff;
+$rim-color: red;
+
+$clock-dial-size: 400px;
+$rim-size: 8px;
+
+$main-bg: aquamarine;
+$clock_background: linear-gradient(90deg, rgba(2,0,36,1) 0%,
+rgba(58,58,168,1) 23%, rgba(0,212,255,1) 56%);
+
+$minutes-count: 1;
+$one-second: 1s;
```

Comment:
u use this var in one place... Do u really need this one?

Reply:
Or rename this to something like that - `$transition-duration`

Reply:
I use them as vars for mixin.
Also i had coment from Oleksii: -Create variables for sizes and durations, now it looks like magic numbers.